langlib/lang.query/src/main/ballerina/helpers.bal

```diff
@@ -146,6 +146,28 @@ function addToTable(stream<Type, CompletionType> strm, table<map<Type>> tbl, err
     return tbl;
 }
 
+function addToMap(stream<Type, CompletionType> strm, map<Type|error> mp, error? err) returns map<Type|error>|error {
```

Comment:
What's err here? Shall we may be add a comment

Reply:
`Type` is `any|error` isn't it? So do we need to have `map<Type|error>` explicitly instead of `map<Type>`

Reply:
> What's err here? Shall we may be add a comment

`err` is used to pass the expression of `on conflict` clause. Sure will add a comment. Thanks.

> Type is any|error isn't it? So do we need to have map<Type|error> explicitly instead of map<Type>

My bad. Will change it. Thanks.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/36366/commits/b0b840abe9bafbbe8bb64ab2b88f1698ac79e89a